api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/specialClinics/SpecialClinicsReferredFromDataEvaluator.java

```diff
@@ -42,7 +42,7 @@ public EvaluatedEncounterData evaluate(EncounterDataDefinition definition, Evalu
         String qry = "select v.encounter_id,\n" +
                 "(case v.referred_from when 1000479 then 'CHU' when 159928 then 'School' when 1000498 then 'Within' when 1175 then 'Not Applicable(N/A)' when 164407 then 'Other facility' else '' end) as referred_from\n" +
                 "from kenyaemr_etl.etl_special_clinics v\n" +
-                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic = :specialClinic;";
+                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic_form_uuid = :specialClinic;";
```

Comment:
Special clinic form uuid is a String. Ensure the parameter value is in quotes, preferrably single quotes.